flink-table-store-core/src/main/java/org/apache/flink/table/store/file/WriteMode.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.store.file;
+
+/** Defines the write mode for flink table store. */
+public enum WriteMode {
+    APPEND_ONLY(
+            "append-only",
+            "The table can only accept append-only insert operations. All rows will be "
+                    + "inserted into the table store without any deduplication or primary/unique key constraint."),
```

Comment:
here is `or` or `and`?

Reply:
The correct sentence is : `Neither data deduplication nor any primary key constraints will be done when inserting rows into table store.`